appium/webdriver/common/appiumby.py

```diff
@@ -25,3 +25,8 @@ class AppiumBy(By):
     ACCESSIBILITY_ID = 'accessibility id'
     IMAGE = '-image'
     CUSTOM = '-custom'
+    FLUTTER_INTEGRATION_SEMANTICS_LABEL = '-flutter semantics label'
```

Comment:
👍 

Can you leave a link as a comment for those as not in appium org repo for future maintenance?

Reply:
Thanks. Could you please clarify link I should add in comment .

Reply:
I think the driver project's GitHub repo is good

Reply:
done